lib/zipkin-tracer/config.rb

```diff
@@ -8,7 +8,8 @@ class Config
     attr_reader :service_name, :service_port, :json_api_host,
```

Comment:
any way to test this stuff? https://github.com/openzipkin/zipkin-ruby/pull/61 for related work